runner/src/main/kotlin/pl/edu/prz/kod/runner/application/Application.kt

```diff
@@ -19,7 +19,7 @@ fun main() {
 
     HttpHandler()
         .tracingHandler
-        .asServer(CustomNetty(port = httpPort))
+        .asServer(pl.edu.prz.kod.runner.adapters.http.CustomUndertow(port = httpPort))
```

Comment:
The `CustomUndertow` could be imported instead of specifying full package name here.